.travis.yml

```diff
@@ -1,8 +1,8 @@
 language: node_js
 node_js:
-  - 8
-  - 9
   - 10
+  - 12
+  - 14
```

Comment:
Felt like v8 support isn't really 2021 thing anymore.

Reply:
It runs in Node 8. That's where I run it atm. :)

Reply:
Hehe, fair enough. We just don't need to test & keep it functional for it (unless someone really wants to!)